package.json

```diff
@@ -14,9 +14,9 @@
     "babel-plugin-transform-runtime": "6.23.0",
     "babel-preset-env": "1.7.0",
     "concurrently": "3.6.0",
-    "fable-loader": "1.1.6",
-    "fable-utils": "1.0.6",
-    "webpack": "4.15.1",
+    "fable-loader": "^2.0.0-alpha-003",
+    "uglifyjs-webpack-plugin": "^1.2.7",
```

Comment:
why do we need uglify?

Reply:
Seems like it is causing some problem with inline method.

@alfonsogarciacaro Will be able to tell us if it's still the case or not.

Reply:
https://github.com/SAFE-Stack/SAFE-BookStore/pull/349/files#diff-19e2e2ba7026ac670b56c4cb0781b7fbR79